### ISSUE TITLE
Issue in migration from 2.12.0 to 3.0.1

### ISSUE THREAD

**Issue by pushp1010** [docusign/native-ios-sdk#137]
After upgrading from 2.12.0 to 3.0.1  following function call started to fail with below error:

▿ Optional<Error>
  - some : Error Domain=envelope.resume.error Code=1014 "Envelope is ready for sync and can not be resumed for signing. " UserInfo={NSLocalizedDescription=Envelope is ready for sync and can not be resumed for signing. , NSLocalizedFailureReason=ENVELOPE_RESUME_FAILED}

```
envelopManager.resumeSigningEnvelope(
            withPresenting: source,
            envelopeId: envelopeID
        ) { [weak self] (_, error) in
            if error != nil {
                self?.onFailed?()
            }
        }
```
It was working fine on previous version. Please suggest.

**Comment by mmohareb** [comment]
Hello @pushp1010 
We have an internal open issue with the same scenario, will keep you posted when we have a fix for this issue

**Comment by mmohareb** [comment]
We are in the process of testing a fix for this issue, we are expecting to release it early next week.

**Comment by mmohareb** [comment]
Hello @pushp1010 
Please refer to release [3.0.2](https://github.com/docusign/native-ios-sdk/releases/tag/v3.0.2) to address this issue

**Comment by mmohareb** [comment]
issue fixed in 3.0.2

**Comment by pushp1010** [comment]
@mmohareb  still getting the same error with 3.0.3 upgrade
